examples/poisson-solver/poisson-solver.cpp

```diff
@@ -62,13 +63,14 @@ void generate_stencil_matrix(gko::matrix::Csr<> *matrix)
 
 
 // Generates the RHS vector given `f` and the boundary conditions.
-template <typename Closure>
-void generate_rhs(Closure f, double u0, double u1, gko::matrix::Dense<> *rhs)
+template <typename Closure, typename ValueType>
+void generate_rhs(Closure f, ValueType u0, ValueType u1,
+                  gko::matrix::Dense<ValueType> *rhs)
 {
     const auto discretization_points = rhs->get_size()[0];
     auto values = rhs->get_values();
     const auto h = 1.0 / (discretization_points + 1);
-    for (int i = 0; i < discretization_points; ++i) {
+    for (size_type i = 0; i < discretization_points; ++i) {
```

Comment:
My bad, this should probably be 
```suggestion
    for (gko::size_type i = 0; i < discretization_points; ++i) {
```

Reply:
Sorry, I was too lazy :)